src/lib/components/comparisonTable/style.module.scss

```diff
@@ -12,6 +12,20 @@
   }
 }
 
+.noScrollBars {
+  // Disable the scrollbar in all browsers
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
+  &::-webkit-scrollbar {
+    /* WebKit */
+    width: 0;
+    height: 0;
+  }
```

Comment:
Don't we need horizontal scroll bars?

Reply:
It's limited to the table content, it does not disable scroll bars globally.